src/wrappers/device.rs

```diff
@@ -12,32 +12,59 @@ pub enum Device {
 /// Cuda related helper functions.
 pub enum Cuda {}
 impl Cuda {
-    /// Returns the number of GPU that can be used.
+    /// Returns the number of CUDA enabled GPUs available.
```

Comment:
Maybe we should re-use the same help text as what appears on the C++ side, e.g. [here](https://github.com/pytorch/pytorch/blob/master/torch/csrc/api/include/torch/cuda.h#L11)?

Also having the documentation duplicated 3 times doesn't sound very nice so I would be in favor of not having it for the C++ side and `torch-sys` (these are only used as internal dependencies anyway).

Reply:
I agree on using the official documentation, I will solve that.

It does not look nice, but for me, it makes it easier to maintain the code when more people are collaborating on it

Reply:
Just to mention that I'm waiting on this one to be fixed before merging the PR, the rest looks all good to me.

Reply:
Should be solved now, I also updated it to use the new changelog